setup.py

```diff
@@ -49,10 +49,14 @@ def read_content(filepath):
     "Programming Language :: Python :: Implementation :: PyPy",
 ]
 
-
 long_description = read_content("README.rst") + read_content(os.path.join("docs/source", "CHANGELOG.rst"))
 
-requires = ["setuptools", "typeguard==2.9.1", "pyspark==3.2.2", "findspark", "pandas"]
+requires = [
+  "typeguard==2.9.1",
+  "pyspark==3.2.2",
```

Comment:
Version is not in sync with SeQuiLa (latest release relies on Spark 3.4.x)- just wondering if we could add some constraints like >=3.4 <3.5 etc instead of hardcoding it here - semanting versioning is respected in the Spark community so any breaking changes shouldn't be introduced if only minor version changes.

Reply:
We can add such constraints in Poetry - I wanted to switch to Poetry in step 2, after verifying that everything works, with publishing, with setup.py build system.

I tried to update to Spark 3.4.4 or 3.4.3 and then use https://mvnrepository.com/artifact/org.biodatageeks/sequila_2.12/1.3.6 but this doesn't work. Before tracking down what is the reason for those failures I wanted to first publish a working version with pyspark==3.2.2 and sequila _2.12:1.1.0

Reply:
Again the same as previously - pls report that as an issue that upgrade to 3.4.x is not possible now